README.md

```diff
@@ -70,6 +70,42 @@ new RollbarConfig
 </dt>
 <dd>A string URI which will be used as the WebClient proxy by passing to the WebProxy constructor.
 </dd>
+<dt>CheckIgnore
+</dt>
+<dd>Function called before sending payload to Rollbar. Return `true` to stop the error from being sent to Rollbar.
+</dd>
+<dt>Truncate
+</dt>
+<dd>Truncates the payload before sending it to Rollbar.
+</dd>
+<dt>Server
+</dt>
+<dd>The server name, as a string.
```

Comment:
I'm unclear on if this is accurate or not.

Reply:
I am not sure where this one comes from. We have complex DTO type called Server that has following properties:
        public static class ReservedProperties
        {
            /// <summary>
            /// The host
            /// </summary>
            public const string Host = "host";
            /// <summary>
            /// The root
            /// </summary>
            public const string Root = "root";
            /// <summary>
            /// The branch
            /// </summary>
            public const string Branch = "branch";
            /// <summary>
            /// The code version
            /// </summary>
            public const string CodeVersion = "code_version";
        }